src/Task.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App;
+
+use Bref\Context\Context;
+use Bref\Event\Handler;
+
+class Task implements Handler
+{
+    public function handle(
+        $event,
```

Comment:
DISCUSS: Why no type hint?

Reply:
We can add, but it'll be 'mixed' because the parent handler has the same. I thought to make an another wrapper to assume we always will have json data but couldn't decide is it a good idea.

Reply:
I don't understand what's going on here.

If we do nothing what is $event here currently?

Reply:
Under the hood, the event is just a result of `json_decode('request body', true);` so that's why it's mixed.

Reply:
Oh hah ok.  I forgot that returns mixed.